### DISCUSSION TITLE
Sharing a parser implementation

### DISCUSSION THREAD

**Discussion by jack-temko** [cooklang/spec#77] [Q&A] [answered]
What is the process for getting a parser implementation listed on the website? Would it also be possible to get it added to the org? The parser in question: https://github.com/ThatTSGuy/cooklang-ts

Thanks in advance!

**Comment by dubadub** [comment, marked as answer]
As discussed on Discord, created a repo https://github.com/cooklang/cooklang-ts. I'll update the website in a bit.

**Comment by dubadub** [comment]
Closing as resolved
